pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -173,11 +171,14 @@ export default function HandlerForm({id, endpointId}) {
       .then((response) => response.json())
       .then((data) => {
         setHandler(data)
-        method === 'POST' && navigate(`/endpoints/${endpointId}`)
+        method === 'POST' && navigate(`/endpoints/${endpointId}`) && setAlert({
+          message: 'Saved handler',
+          type: 'success'
+        })
+        setAlert({message: 'Updated handler', type: 'success'})
```

Comment:
I think, to be more consistent with the rest of our codebase, we should be checking on if `id` was set or not. Instead of this `=== 'POST'` check. 

If `id` is set, it was a `Update`, otherwise it was a `Create`.

Might even be something like:

```
setAlert({
  message: '${id ? "Updated" : "Created"} handler',
  type: ...
})
```

Reply:
Note: above syntax will automatically no longer be needed when we implement the `ApiService` here (because we've decoupled the `create` and `update` functions).